programs/single_linked_lists.py

```diff
@@ -0,0 +1,58 @@
+class Node:
+    def __init__(self, data):
+        self.data = data
+        self.next = None
```

Comment:
Implement `__str__` function here that prints the whole list recursively. Avoid using a `SingleLinkedList` class for interview problems